### DISCUSSION TITLE
How do I save a gt docx table from within Quarto?

### DISCUSSION THREAD

**Discussion by robertprince** [quarto-dev/quarto-cli#11353] [Q&A] [answered]
### Description

I wish to save a gt docx table from Quarto and am receiving the following error message:

Pandoc can convert to DOCX, but not to DOC.
Error:
! pandoc document conversion failed with error 22

My qmd file is:

````
---
title: "Trial3"
format: html
editor: visual
---

{r}
library(gt)

tab_1 <-
  gtcars |>
  dplyr::select(model, year, hp, trq) |>
  dplyr::slice(1:5) |>
  gt(rowname_col = "model") |>
  tab_stubhead(label = "car")

tab_1

tab_1 |> gtsave("tab_1.docx")
````

I am using Quarto version 1.6.32 on a Windows 10 pc

**Comment by mcanouil** [comment, marked as answer]
Why are you opening a new discussion?

- https://github.com/quarto-dev/quarto-cli/discussions/11352#discussioncomment-11195426

Please edit your previous issue and properly format your post as shown.
